custom/ewsghana/api.py

```diff
@@ -658,6 +658,9 @@ def web_user_sync(self, ews_webuser):
             if self.domain not in user.get_domains():
                 user.add_domain_membership(self.domain, location_id=location_id)
 
+            # We are migrating only active users
+            user.is_active = True
```

Comment:
ok - does anything need to be rerun because of this (does the whole migration have to rerun from the start)?